### PR TITLE
support added for OCP-37098

### PR DESCRIPTION
**Pull request by yapei**
@yanpzhan  Ready for your review, PTAL thanks

### CODE REVIEW

File: lib/openshift/catalog_source.rb [hunk 1]
```diff
@@ -9,5 +9,30 @@ def ready?(user:, quiet: false, cached: false)
...
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      return rr['spec'].has_key?('publisher') ? rr.dig('spec', 'publisher') : '-'	
+  	end
+
+  	def status(user: nil, quiet: false, cached: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      return rr.dig('status', 'connectionState', 'lastObservedState')  		
+  	end
+
+  	def displayname(user: nil, quiet: false, cached: false)
```
Comment (liangxia) [inline lib/openshift/catalog_source.rb:33]: I think we can cache most of the resources, except status, by default

Comment (yapei) [reply lib/openshift/catalog_source.rb:33]: @liangxia  Good idea, let me update

Comment (yapei) [reply lib/openshift/catalog_source.rb]: updated, PTAL @liangxia

**Pull request merged**